contracts/minting/Controller.sol

```diff
@@ -50,10 +54,16 @@ contract Controller is Ownable {
     // onlyOwner functions
 
     /**
-     * @dev set the controller of a particular _worker
-     * Argument _worker must not be 0x00, call removeController(_controller) instead.
+     * @dev set the controller of a particular _worker.
+     * Argument _worker must not be 0x00. Call removeController(_controller)
+     * instead to disable the controller.
      */
-    function configureController(address _controller, address _worker) onlyOwner public returns (bool) {
+    function configureController(
+        address _controller,
+        address _worker
+    )
+        onlyOwner public returns (bool)
```

Comment:
Can we put modifiers/return on their own line as recommend by the style guide?
https://solidity.readthedocs.io/en/v0.4.24/style-guide.html#function-declaration